eq.yml

```diff
@@ -0,0 +1,810 @@
+          cp -R go-launch-a-survey/ $HOME/gopath/src/github.com/ONSdigital/
+          cd $HOME/gopath/src/github.com/ONSdigital/go-launch-a-survey
+
+          go get -u github.com/golang/dep/cmd/dep
+          dep ensure
+          docker run --rm -v $(pwd):/src centurylink/golang-builder
+
+          service docker stop
+
+          cp -R . $build_dir/go-launch-a-survey-compiled
+  - put: go-launch-a-survey-image
+    params:
+      build: go-launch-a-survey-compiled
+      tag: go-launch-a-survey/.git/HEAD
+    get_params:
+      skip_download: true
+
+- name: build-eq-survey-register
+  plan:
+  - get: eq-survey-register
+    trigger: true
+  - task: Build
+    privileged: true
+    config:
+      platform: linux
+      image_resource:
+        type: docker-image
+        source:
+          repository: onsdigital/eq-golang-build
+      inputs:
+      - name: eq-survey-register
+      outputs:
+      - name: eq-survey-register-compiled
+      run:
+        path: bash
+        args:
+        - -exc
+        - |
+          service docker start
+
+          build_dir=$(pwd)
+          export GOPATH=$HOME/gopath
+          export PATH=$HOME/gopath/bin:$PATH
+          mkdir -p $HOME/gopath/src/github.com/ONSdigital/eq-survey-register
+          cp -R eq-survey-register/ $HOME/gopath/src/github.com/ONSdigital/
+          cd $HOME/gopath/src/github.com/ONSdigital/eq-survey-register
+
+          go get -u github.com/golang/dep/cmd/dep
+          dep ensure
+          docker run --rm -v $(pwd):/src jonnyshaw89/golang-builder
+
+          service docker stop
+
+          cp -R . $build_dir/eq-survey-register-compiled
+  - put: eq-survey-register-image
+    params:
+      build: eq-survey-register-compiled
+      tag: eq-survey-register/.git/HEAD
+    get_params:
+      skip_download: true
+
+- name: build-eq-schema-validator
+  plan:
+  - get: eq-schema-validator
+    trigger: true
+  - task: Run Tests
+    config:
+      platform: linux
+      image_resource:
+        type: docker-image
+        source:
+          repository: onsdigital/eq-python-build
+      inputs:
+      - name: eq-schema-validator
+      outputs:
+      - name: compiled-eq-schema-validator
+      run:
+        path: sh
+        args:
+        - -exc
+        - |
+          pyenv install 3.6.3
+
+          cd eq-schema-validator
+          pip install -U pip pipenv
+          pipenv install --dev --deploy
+          pipenv run ./scripts/build.sh
+
+          cp -R ../eq-schema-validator/* ../compiled-eq-schema-validator
+          cp ../eq-schema-validator/.git/HEAD ../compiled-eq-schema-validator/.application-version
+  - put: eq-schema-validator-image
+    params:
+      build: compiled-eq-schema-validator
+      tag: eq-schema-validator/.git/HEAD
+    get_params:
+      skip_download: true
+
+- name: staging-deploy
+  serial_groups: [staging-deploy, staging-functional-tests-core, staging-functional-tests-ukis, staging-functional-tests-census, staging-destroy]
+  plan:
```

Comment:
@jonnyshaw89 dont we need to add `eq-smoke-test` here?

Reply:
Not until its merged. that can go in as a later PR